grassroots-backend/src/grassroots-shared/Contact.dto.ts

```diff
@@ -78,37 +62,34 @@ export class ContactSearchInDTO {
   @IsInt()
   @Min(1)
   id?: number;
-  @IsOptional()
-  email?: string;
-  @IsOptional()
-  firstName?: string;
-  @IsOptional()
-  lastName?: string;
-  @IsOptional()
-  phoneNumber?: string;
 }
 
-export class PaginatedContactSearchInDTO {
+export class ContactSearchRequestDTO extends IntersectionType(
+  OmitType(PartialType(ContactDTO), ["id"]),
+  ContactSearchRequestId,
+) {}
```

Comment:
My understanding is that we need the decorators on these classes for validation. With all the type modifiers like `IntersectionType`, do the decorators exist on the resulting instance?